com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -1782,6 +1782,82 @@ def flatten_dict(tpl) -> Iterable[SampleSchema]:
         oport = op.addOutputPort(name=_name, schema=schema)
         return Stream(self.topology, oport)._make_placeable()._layout('FlatMap', name=op.runtime_id, orig_name=name)._add_hints(hints)
     
+    def catch_exceptions(self, exception_type:str='streams', tuple_trace:bool=False, stack_trace:bool=False):
+        """ When applied to a primitive operator, exceptions of the specified type that are thrown by the operator while processing a tuple are caught. 
+
+        This functions adds the `@catch annotation <https://www.ibm.com/support/knowledgecenter/SSCRJU_4.3.0/com.ibm.streams.ref.doc/doc/catchannotation.html>`_ on an operator.
```

Comment:
I would not add a reference to the SPL documentation in the KC here as the function is good enough described in the first sentence (even if the first sentence is guite long and nested) and below.